### ISSUE TITLE
Restoring a Backup deletes all backups and fails ... Basicly Azuracast kills itself

### ISSUE THREAD

**Issue by Schrebergarten03-links** [AzuraCast/AzuraCast#5958]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### What happened?

i use a cifs mounted nas as library 👍 

with docker compose override:
 services:
  web:
    volumes:
           - /mnt/music/xxx:/var/azuracast/stations/xxx/media/  

had to reboot server ... azuracast restarted didnt find the media folders in time ... and all files were gone from the database ... media of the station had 0 files ... 

fixed the cifs mount and restarted azuracast docker ... 0 files again ... had no nerve to rescan library and redoo alllllllllll playlists ... 

So i wanted to reload a backup ---- that fixed it for me once

First problem backups arent under /var/azuracast/automatic_backup_xxx like shown in the GUI ... 
they are saved under /var/lib/docker/volumes/azuracast_backups/_data/ ...

so i did the bad thing ... 

./docker.sh restore /var/lib/docker/volumes/azuracast_backups/_data/automatic_backup_20221221_020621.zip

aaaaaand everything is gone ... first step website is greeting me ...

the backup deleted also every other backup in /var/lib/docker/volumes/azuracast_backups/_data/ ...

so ... i have to start over ... for the 3 rd time this happend now ... 
So i thought better post this ...

This is a pretty nice clusterfuck.

Dunno why azuracast uses /var/lib/docker/volumes/ instead of /var/azuracast/
might be a misconfiguration by me ... but i just followed the installation guide ...

Well gonna rebuild my 5 radiostations now ... ~2 months of work gone ... yes it hurts.

### Relevant log output

```shell
root:/var/azuracast# ./docker.sh restore /var/lib/docker/volumes/azuracast_backups/_data/aut                                                                                                             omatic_backup_20221221_020621.zip
Restoring will remove any existing AzuraCast installation data, replacing it with your backup. Continu                                                                                                             e? [Y/n] Y
[+] Running 2/2
 ⠿ Container azuracast        Removed                                                           10.6s
 ⠿ Network azuracast_default  Removed                                                            0.1s
azuracast_backups
azuracast_db_data
azuracast_sftpgo_data
azuracast_www_uploads
azuracast_www_vendor
azuracast_backups
mv: can't rename '/backup_src/automatic_backup_20221221_020621.zip': No such file or directory
WARN[0000] volume "azuracast_backups" already exists but was not created by Docker Compose. Use `exter                                                                                                             nal: true` to use an existing volume
[+] Running 5/0
 ⠿ Network azuracast_default       Created                                                       0.0s
 ⠿ Volume "azuracast_sftpgo_data"  Created                                                       0.0s
 ⠿ Volume "azuracast_www_vendor"   Created                                                       0.0s
 ⠿ Volume "azuracast_db_data"      Created                                                       0.0s
 ⠿ Volume "azuracast_www_uploads"  Created                                                       0.0s
** Running startup script '/etc/my_init.d/00_disable_mariadb.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/00_disable_redis.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/00_setup_user.sh'...
usermod: no changes
Docker 'azuracast' User UID: 1000
Docker 'azuracast' User GID: 1000
** Startup script complete.
** Running startup script '/etc/my_init.d/01_self_signed_ssl.sh'...
Generating self-signed certificate...
.....+...+......+.+.....+.+.....+...+....+.....+...+...+.......+...+.....+.......+..+.......+.........                                                                                                             +.....+.......+...............+..+.+.....+.+.....+...+...+....+.....+...+...+.......+........+.......+                                                                                                             ......+......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*......+........+......+                                                                                                             ..................+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*...+..+....+......                                                                                                             ..+......+.+......+..................+...........+.+..+.......++++++++++++++++++++++++++++++++++++++++                                                                                                             +++++++++++++++++++++++++
....+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.........+......+........+..                                                                                                             .+.......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*...+......+......+......+..                                                                                                             ...............+...+...+.+...........+...+....+..+....+..+.......+.....+.............+++++++++++++++++                                                                                                             ++++++++++++++++++++++++++++++++++++++++++++++++
-----
** Startup script complete.
** Running startup script '/etc/my_init.d/02_prepare_cron.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/04_mariadb_conf.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/05_nginx_conf.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/05_setup_db.sh'...
2022-12-21 22:29:32+00:00 [Note] [Entrypoint]: Initial DB setup...
2022-12-21 22:29:32+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
2022-12-21 22:29:32+00:00 [Note] [Entrypoint]: Initial DB setup...
2022-12-21 22:29:32+00:00 [Note] [Entrypoint]: Initializing database files


PLEASE REMEMBER TO SET A PASSWORD FOR THE MariaDB root USER !
To do so, start the server, then issue the following command:

'/usr/bin/mysql_secure_installation'

which will also give you the option of removing the test
databases and anonymous user created by default.  This is
strongly recommended for production servers.

See the MariaDB Knowledgebase at https://mariadb.com/kb

Please report any problems at https://mariadb.org/jira

The latest information about MariaDB is available at https://mariadb.org/.

Consider joining MariaDB's strong and vibrant community:
https://mariadb.org/get-involved/

2022-12-21 22:29:35+00:00 [Note] [Entrypoint]: Database files initialized
2022-12-21 22:29:35+00:00 [Note] [Entrypoint]: Starting temporary server
2022-12-21 22:29:35+00:00 [Note] [Entrypoint]: Waiting for server startup
2022-12-21 22:29:35 0 [Note] mysqld (server 10.9.3-MariaDB-1:10.9.3+maria~ubu2204) starting as process                                                                                                              139 ...
2022-12-21 22:29:35 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
2022-12-21 22:29:35 0 [Note] InnoDB: Number of transaction pools: 1
2022-12-21 22:29:35 0 [Note] InnoDB: Using crc32 + pclmulqdq instructions
2022-12-21 22:29:35 0 [Note] mysqld: O_TMPFILE is not supported on /tmp (disabling future attempts)
2022-12-21 22:29:35 0 [Note] InnoDB: Initializing buffer pool, total size = 128.000MiB, chunk size = 2                                                                                                             .000MiB
2022-12-21 22:29:35 0 [Note] InnoDB: Completed initialization of buffer pool
2022-12-21 22:29:35 0 [Note] InnoDB: File system buffers for log disabled (block size=512 bytes)
2022-12-21 22:29:35 0 [Note] InnoDB: 128 rollback segments are active.
2022-12-21 22:29:35 0 [Note] InnoDB: Setting file './ibtmp1' size to 12.000MiB. Physically writing the                                                                                                              file full; Please wait ...
2022-12-21 22:29:35 0 [Note] InnoDB: File './ibtmp1' size is now 12.000MiB.
2022-12-21 22:29:35 0 [Note] InnoDB: log sequence number 46590; transaction id 14
2022-12-21 22:29:35 0 [Note] Plugin 'FEEDBACK' is disabled.
2022-12-21 22:29:35 0 [Warning] 'user' entry 'root@187c13670168' ignored in --skip-name-resolve mode.
2022-12-21 22:29:35 0 [Warning] 'proxies_priv' entry '@% root@187c13670168' ignored in --skip-name-res                                                                                                             olve mode.
2022-12-21 22:29:35 0 [Note] mysqld: ready for connections.
Version: '10.9.3-MariaDB-1:10.9.3+maria~ubu2204'  socket: '/run/mysqld/mysqld.sock'  port: 0  mariadb.                                                                                                             org binary distribution
2022-12-21 22:29:36+00:00 [Note] [Entrypoint]: Temporary server started.
2022-12-21 22:29:37+00:00 [Note] [Entrypoint]: Creating database azuracast
2022-12-21 22:29:37+00:00 [Note] [Entrypoint]: Creating user azuracast
2022-12-21 22:29:37+00:00 [Note] [Entrypoint]: Giving user azuracast access to schema azuracast
2022-12-21 22:29:37+00:00 [Note] [Entrypoint]: Securing system users (equivalent to running mysql_secu                                                                                                             re_installation)

2022-12-21 22:29:37+00:00 [Note] [Entrypoint]: /etc/my_init.d/05_setup_db.sh: running /docker-entrypoi                                                                                                             nt-initdb.d/00-azuracast.sql


2022-12-21 22:29:38+00:00 [Note] [Entrypoint]: Stopping temporary server
2022-12-21 22:29:38 0 [Note] mysqld (initiated by: unknown): Normal shutdown
2022-12-21 22:29:38 0 [Note] InnoDB: FTS optimize thread exiting.
2022-12-21 22:29:38 0 [Note] InnoDB: Starting shutdown...
2022-12-21 22:29:38 0 [Note] InnoDB: Dumping buffer pool(s) to /var/lib/mysql/ib_buffer_pool
2022-12-21 22:29:38 0 [Note] InnoDB: Buffer pool(s) dump completed at 221221 22:29:38
2022-12-21 22:29:38 0 [Note] InnoDB: Removed temporary tablespace data file: "./ibtmp1"
2022-12-21 22:29:38 0 [Note] InnoDB: Shutdown completed; log sequence number 237376; transaction id 13                                                                                                             0
2022-12-21 22:29:38 0 [Note] mysqld: Shutdown complete

2022-12-21 22:29:38+00:00 [Note] [Entrypoint]: Temporary server stopped

2022-12-21 22:29:38+00:00 [Note] [Entrypoint]: MariaDB init process done. Ready for start up.

** Startup script complete.
** Running startup script '/etc/my_init.d/07_sftpgo_conf.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/10_clear_temp.sh'...
** Startup script complete.
** Running Supervisord...
** Supervisord started.
** Running uptime_wait...
Starting up AzuraCast services...
🖥️－🎶－🎵📻 Waiting...Services started up and ready!
** Services up and ready!
This script must be run as the "azuracast" user. Rerunning...
AzuraCast Backup Restore
(Environment: production)
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Lock file operations: 0 installs, 89 updates, 2 removals
  - Removing beberlei/assert (v3.3.2)
  - Removing thecodingmachine/safe (v2.3.0)
  - Upgrading aws/aws-sdk-php (3.235.10 => 3.255.0)
  - Upgrading azuracast/nowplaying (dev-main add6f79 => dev-main 82f6689)
  - Upgrading bacon/bacon-qr-code (2.0.7 => 2.0.8)
  - Upgrading codeception/codeception (5.0.2 => 5.0.5)
  - Upgrading codeception/lib-asserts (2.0.0 => 2.0.1)
  - Upgrading codeception/lib-innerbrowser (3.1.2 => 3.1.3)
  - Upgrading composer/ca-bundle (1.3.3 => 1.3.4)
  - Upgrading doctrine/annotations (1.13.3 => 1.14.2)
  - Upgrading doctrine/collections (1.7.3 => 2.1.1)
  - Upgrading doctrine/common (3.4.0 => 3.4.3)
  - Upgrading doctrine/dbal (3.4.4 => 3.5.2)
  - Upgrading doctrine/event-manager (1.1.2 => 1.2.0)
  - Upgrading doctrine/inflector (2.0.5 => 2.0.6)
  - Upgrading doctrine/orm (2.13.1 => 2.14.0)
  - Upgrading doctrine/persistence (3.0.3 => 3.1.2)
  - Upgrading filp/whoops (2.14.5 => 2.14.6)
  - Upgrading friendsofphp/proxy-manager-lts (v1.0.12 => v1.0.13)
  - Upgrading gettext/languages (2.9.0 => 2.10.0)
  - Upgrading graham-campbell/guzzle-factory (v5.1.1 => v5.1.2)
  - Upgrading guzzlehttp/psr7 (2.4.1 => 2.4.3)
  - Upgrading laminas/laminas-code (4.7.0 => 4.8.0)
  - Upgrading league/flysystem (3.4.0 => 3.12.0)
  - Upgrading league/flysystem-aws-s3-v3 (3.3.0 => 3.10.3)
  - Upgrading league/flysystem-sftp-v3 (3.3.0 => 3.10.3)
  - Upgrading matomo/device-detector (6.0.3 => 6.0.4)
  - Upgrading mezzio/mezzio-session (1.11.0 => 1.12.0)
  - Upgrading mezzio/mezzio-session-cache (1.9.0 => 1.10.0)
  - Upgrading nesbot/carbon (2.62.1 => 2.64.0)
  - Upgrading nikic/php-parser (v4.15.1 => v4.15.2)
  - Upgrading pagerfanta/core (v3.6.2 => v3.7.0)
  - Upgrading pagerfanta/doctrine-collections-adapter (v3.6.2 => v3.7.0)
  - Upgrading pagerfanta/doctrine-orm-adapter (v3.6.2 => v3.7.0)
  - Upgrading php-ffmpeg/php-ffmpeg (v1.0.1 => v1.1.0)
  - Upgrading phpseclib/phpseclib (3.0.16 => 3.0.18)
  - Upgrading phpstan/phpdoc-parser (1.7.0 => 1.15.3)
  - Upgrading phpstan/phpstan (1.8.5 => 1.9.4)
  - Upgrading phpstan/phpstan-doctrine (1.3.13 => 1.3.27)
  - Upgrading phpunit/php-code-coverage (9.2.17 => 9.2.22)
  - Upgrading phpunit/phpunit (9.5.24 => 9.5.27)
  - Upgrading psy/psysh (v0.11.8 => v0.11.9)
  - Upgrading pyrech/composer-changelogs (v1.8.1 => v1.8.2)
  - Upgrading ramsey/uuid (4.5.1 => 4.7.0)
  - Upgrading roave/security-advisories (dev-latest 264a508 => dev-latest 58046a3)
  - Upgrading slevomat/coding-standard (8.4.0 => 8.7.1)
  - Upgrading slim/http (1.2.0 => 1.3)
  - Upgrading slim/slim (4.10.0 => 4.11.0)
  - Upgrading softcreatr/jsonpath (0.8.0 => 0.8.1)
  - Upgrading spatie/dropbox-api (1.20.2 => 1.21.0)
  - Upgrading spomky-labs/otphp (v11.0.1 => 11.1.0)
  - Upgrading supervisorphp/supervisor (dev-main 29b6ecb => dev-main e8a5ac3)
  - Upgrading symfony/beanstalkd-messenger (v6.1.3 => v6.2.0)
  - Upgrading symfony/browser-kit (v6.1.3 => v6.2.0)
  - Upgrading symfony/cache (v6.1.3 => v6.2.2)
  - Upgrading symfony/cache-contracts (v3.1.1 => v3.2.0)
  - Upgrading symfony/console (v6.1.4 => v6.2.2)
  - Upgrading symfony/css-selector (v6.1.3 => v6.2.0)
  - Upgrading symfony/deprecation-contracts (v3.1.1 => v3.2.0)
  - Upgrading symfony/dom-crawler (v6.1.4 => v6.2.0)
  - Upgrading symfony/event-dispatcher (v6.1.0 => v6.2.2)
  - Upgrading symfony/event-dispatcher-contracts (v3.1.1 => v3.2.0)
  - Upgrading symfony/expression-language (v6.1.3 => v6.2.2)
  - Upgrading symfony/filesystem (v6.1.4 => v6.2.0)
  - Upgrading symfony/finder (v6.1.3 => v6.2.0)
  - Upgrading symfony/intl (v6.1.0 => v6.2.0)
  - Upgrading symfony/lock (v6.1.3 => v6.2.0)
  - Upgrading symfony/mailer (v6.1.4 => v6.2.2)
  - Upgrading symfony/messenger (v6.1.3 => v6.2.2)
  - Upgrading symfony/mime (v6.1.4 => v6.2.2)
  - Upgrading symfony/options-resolver (v6.1.0 => v6.2.0)
  - Upgrading symfony/polyfill-ctype (v1.26.0 => v1.27.0)
  - Upgrading symfony/polyfill-intl-idn (v1.26.0 => v1.27.0)
  - Upgrading symfony/process (v6.1.3 => v6.2.0)
  - Upgrading symfony/property-access (v6.1.3 => v6.2.0)
  - Upgrading symfony/property-info (v6.1.3 => v6.2.0)
  - Upgrading symfony/rate-limiter (v6.1.3 => v6.2.2)
  - Upgrading symfony/redis-messenger (v6.1.3 => v6.2.0)
  - Upgrading symfony/serializer (v6.1.4 => v6.2.2)
  - Upgrading symfony/service-contracts (v3.1.1 => v3.2.0)
  - Upgrading symfony/stopwatch (v6.1.0 => v6.2.0)
  - Upgrading symfony/string (v6.1.4 => v6.2.2)
  - Upgrading symfony/translation (v6.1.4 => v6.2.2)
  - Upgrading symfony/translation-contracts (v3.1.1 => v3.2.0)
  - Upgrading symfony/validator (v6.1.4 => v6.2.2)
  - Upgrading symfony/var-dumper (v6.1.3 => v6.2.2)
  - Upgrading symfony/var-exporter (v6.1.3 => v6.2.2)
  - Upgrading symfony/yaml (v6.1.4 => v6.2.2)
  - Upgrading vlucas/phpdotenv (v5.4.1 => v5.5.0)
  - Upgrading voku/portable-utf8 (6.0.9 => 6.0.11)
  - Upgrading zircote/swagger-php (4.4.9 => 4.5.3)
Writing lock file
Installing dependencies from lock file
Package operations: 0 installs, 71 updates, 2 removals
  - Downloading symfony/polyfill-ctype (v1.27.0)
  - Downloading symfony/string (v6.2.2)
  - Downloading symfony/service-contracts (v3.2.0)
  - Downloading symfony/deprecation-contracts (v3.2.0)
  - Downloading symfony/console (v6.2.2)
  - Downloading doctrine/event-manager (1.2.0)
  - Downloading doctrine/persistence (3.1.2)
  - Downloading doctrine/inflector (2.0.6)
  - Downloading doctrine/dbal (3.5.2)
  - Downloading doctrine/common (3.4.3)
  - Downloading doctrine/collections (2.1.1)
  - Downloading doctrine/orm (2.14.0)
  - Downloading symfony/serializer (v6.2.2)
  - Downloading league/flysystem (3.12.0)
  - Downloading guzzlehttp/psr7 (2.4.3)
  - Downloading azuracast/nowplaying (dev-main 82f6689)
  - Downloading symfony/event-dispatcher-contracts (v3.2.0)
  - Downloading symfony/event-dispatcher (v6.2.2)
  - Downloading slim/slim (4.11.0)
  - Downloading bacon/bacon-qr-code (2.0.8)
  - Downloading composer/ca-bundle (1.3.4)
  - Downloading symfony/stopwatch (v6.2.0)
  - Downloading symfony/filesystem (v6.2.0)
  - Downloading laminas/laminas-code (4.8.0)
  - Downloading friendsofphp/proxy-manager-lts (v1.0.13)
  - Downloading gettext/languages (2.10.0)
  - Downloading nikic/php-parser (v4.15.2)
  - Downloading graham-campbell/guzzle-factory (v5.1.2)
  - Downloading aws/aws-sdk-php (3.255.0)
  - Downloading league/flysystem-aws-s3-v3 (3.10.3)
  - Downloading phpseclib/phpseclib (3.0.18)
  - Downloading league/flysystem-sftp-v3 (3.10.3)
  - Downloading matomo/device-detector (6.0.4)
  - Downloading mezzio/mezzio-session (1.12.0)
  - Downloading mezzio/mezzio-session-cache (1.10.0)
  - Downloading symfony/translation-contracts (v3.2.0)
  - Downloading symfony/translation (v6.2.2)
  - Downloading nesbot/carbon (2.64.0)
  - Downloading pagerfanta/core (v3.7.0)
  - Downloading pagerfanta/doctrine-collections-adapter (v3.7.0)
  - Downloading pagerfanta/doctrine-orm-adapter (v3.7.0)
  - Downloading symfony/process (v6.2.0)
  - Downloading symfony/var-exporter (v6.2.2)
  - Downloading symfony/cache-contracts (v3.2.0)
  - Downloading symfony/cache (v6.2.2)
  - Downloading php-ffmpeg/php-ffmpeg (v1.1.0)
  - Downloading symfony/expression-language (v6.2.2)
  - Downloading ramsey/uuid (4.7.0)
  - Downloading slim/http (1.3)
  - Downloading spatie/dropbox-api (1.21.0)
  - Downloading spomky-labs/otphp (11.1.0)
  - Downloading supervisorphp/supervisor (dev-main e8a5ac3)
  - Downloading symfony/messenger (v6.2.2)
  - Downloading symfony/beanstalkd-messenger (v6.2.0)
  - Downloading symfony/intl (v6.2.0)
  - Downloading symfony/lock (v6.2.0)
  - Downloading symfony/polyfill-intl-idn (v1.27.0)
  - Downloading symfony/mime (v6.2.2)
  - Downloading symfony/mailer (v6.2.2)
  - Downloading symfony/property-info (v6.2.0)
  - Downloading symfony/property-access (v6.2.0)
  - Downloading symfony/options-resolver (v6.2.0)
  - Downloading symfony/rate-limiter (v6.2.2)
  - Downloading symfony/redis-messenger (v6.2.0)
  - Downloading symfony/validator (v6.2.2)
  - Downloading vlucas/phpdotenv (v5.5.0)
  - Downloading voku/portable-utf8 (6.0.11)
  - Downloading symfony/yaml (v6.2.2)
  - Downloading symfony/finder (v6.2.0)
  - Downloading doctrine/annotations (1.14.2)
  - Downloading zircote/swagger-php (4.5.3)
  - Removing thecodingmachine/safe (v2.3.0)
  - Removing beberlei/assert (v3.3.2)
  - Upgrading symfony/polyfill-ctype (v1.26.0 => v1.27.0): Extracting archive
  - Upgrading symfony/string (v6.1.4 => v6.2.2): Extracting archive
  - Upgrading symfony/service-contracts (v3.1.1 => v3.2.0): Extracting archive
  - Upgrading symfony/deprecation-contracts (v3.1.1 => v3.2.0): Extracting archive
  - Upgrading symfony/console (v6.1.4 => v6.2.2): Extracting archive
  - Upgrading doctrine/event-manager (1.1.2 => 1.2.0): Extracting archive
  - Upgrading doctrine/persistence (3.0.3 => 3.1.2): Extracting archive
  - Upgrading doctrine/inflector (2.0.5 => 2.0.6): Extracting archive
  - Upgrading doctrine/dbal (3.4.4 => 3.5.2): Extracting archive
  - Upgrading doctrine/common (3.4.0 => 3.4.3): Extracting archive
  - Upgrading doctrine/collections (1.7.3 => 2.1.1): Extracting archive
  - Upgrading doctrine/orm (2.13.1 => 2.14.0): Extracting archive
  - Upgrading symfony/serializer (v6.1.4 => v6.2.2): Extracting archive
  - Upgrading league/flysystem (3.4.0 => 3.12.0): Extracting archive
  - Upgrading guzzlehttp/psr7 (2.4.1 => 2.4.3): Extracting archive
  - Upgrading azuracast/nowplaying (dev-main add6f79 => dev-main 82f6689): Extracting archive
  - Upgrading symfony/event-dispatcher-contracts (v3.1.1 => v3.2.0): Extracting archive
  - Upgrading symfony/event-dispatcher (v6.1.0 => v6.2.2): Extracting archive
  - Upgrading slim/slim (4.10.0 => 4.11.0): Extracting archive
  - Upgrading bacon/bacon-qr-code (2.0.7 => 2.0.8): Extracting archive
  - Upgrading composer/ca-bundle (1.3.3 => 1.3.4): Extracting archive
  - Upgrading symfony/stopwatch (v6.1.0 => v6.2.0): Extracting archive
  - Upgrading symfony/filesystem (v6.1.4 => v6.2.0): Extracting archive
  - Upgrading laminas/laminas-code (4.7.0 => 4.8.0): Extracting archive
  - Upgrading friendsofphp/proxy-manager-lts (v1.0.12 => v1.0.13): Extracting archive
  - Upgrading gettext/languages (2.9.0 => 2.10.0): Extracting archive
  - Upgrading nikic/php-parser (v4.15.1 => v4.15.2): Extracting archive
  - Upgrading graham-campbell/guzzle-factory (v5.1.1 => v5.1.2): Extracting archive
  - Upgrading aws/aws-sdk-php (3.235.10 => 3.255.0): Extracting archive
  - Upgrading league/flysystem-aws-s3-v3 (3.3.0 => 3.10.3): Extracting archive
  - Upgrading phpseclib/phpseclib (3.0.16 => 3.0.18): Extracting archive
  - Upgrading league/flysystem-sftp-v3 (3.3.0 => 3.10.3): Extracting archive
  - Upgrading matomo/device-detector (6.0.3 => 6.0.4): Extracting archive
  - Upgrading mezzio/mezzio-session (1.11.0 => 1.12.0): Extracting archive
  - Upgrading mezzio/mezzio-session-cache (1.9.0 => 1.10.0): Extracting archive
  - Upgrading symfony/translation-contracts (v3.1.1 => v3.2.0): Extracting archive
  - Upgrading symfony/translation (v6.1.4 => v6.2.2): Extracting archive
  - Upgrading nesbot/carbon (2.62.1 => 2.64.0): Extracting archive
  - Upgrading pagerfanta/core (v3.6.2 => v3.7.0): Extracting archive
  - Upgrading pagerfanta/doctrine-collections-adapter (v3.6.2 => v3.7.0): Extracting archive
  - Upgrading pagerfanta/doctrine-orm-adapter (v3.6.2 => v3.7.0): Extracting archive
  - Upgrading symfony/process (v6.1.3 => v6.2.0): Extracting archive
  - Upgrading symfony/var-exporter (v6.1.3 => v6.2.2): Extracting archive
  - Upgrading symfony/cache-contracts (v3.1.1 => v3.2.0): Extracting archive
  - Upgrading symfony/cache (v6.1.3 => v6.2.2): Extracting archive
  - Upgrading php-ffmpeg/php-ffmpeg (v1.0.1 => v1.1.0): Extracting archive
  - Upgrading symfony/expression-language (v6.1.3 => v6.2.2): Extracting archive
  - Upgrading ramsey/uuid (4.5.1 => 4.7.0): Extracting archive
  - Upgrading slim/http (1.2.0 => 1.3): Extracting archive
  - Upgrading spatie/dropbox-api (1.20.2 => 1.21.0): Extracting archive
  - Upgrading spomky-labs/otphp (v11.0.1 => 11.1.0): Extracting archive
  - Upgrading supervisorphp/supervisor (dev-main 29b6ecb => dev-main e8a5ac3): Extracting archive
  - Upgrading symfony/messenger (v6.1.3 => v6.2.2): Extracting archive
  - Upgrading symfony/beanstalkd-messenger (v6.1.3 => v6.2.0): Extracting archive
  - Upgrading symfony/intl (v6.1.0 => v6.2.0): Extracting archive
  - Upgrading symfony/lock (v6.1.3 => v6.2.0): Extracting archive
  - Upgrading symfony/polyfill-intl-idn (v1.26.0 => v1.27.0): Extracting archive
  - Upgrading symfony/mime (v6.1.4 => v6.2.2): Extracting archive
  - Upgrading symfony/mailer (v6.1.4 => v6.2.2): Extracting archive
  - Upgrading symfony/property-info (v6.1.3 => v6.2.0): Extracting archive
  - Upgrading symfony/property-access (v6.1.3 => v6.2.0): Extracting archive
  - Upgrading symfony/options-resolver (v6.1.0 => v6.2.0): Extracting archive
  - Upgrading symfony/rate-limiter (v6.1.3 => v6.2.2): Extracting archive
  - Upgrading symfony/redis-messenger (v6.1.3 => v6.2.0): Extracting archive
  - Upgrading symfony/validator (v6.1.4 => v6.2.2): Extracting archive
  - Upgrading vlucas/phpdotenv (v5.4.1 => v5.5.0): Extracting archive
  - Upgrading voku/portable-utf8 (6.0.9 => 6.0.11): Extracting archive
  - Upgrading symfony/yaml (v6.1.4 => v6.2.2): Extracting archive
  - Upgrading symfony/finder (v6.1.3 => v6.2.0): Extracting archive
  - Upgrading doctrine/annotations (1.13.3 => 1.14.2): Extracting archive
  - Upgrading zircote/swagger-php (4.4.9 => 4.5.3): Extracting archive
Package azuracast/doctrine-batch-utilities is abandoned, you should avoid using it. No replacement was                                                                                                              suggested.
Package azuracast/doctrine-entity-normalizer is abandoned, you should avoid using it. No replacement w                                                                                                             as suggested.
Package azuracast/flysystem-v2-extensions is abandoned, you should avoid using it. No replacement was                                                                                                              suggested.
Package azuracast/slim-callable-eventdispatcher is abandoned, you should avoid using it. No replacemen                                                                                                             t was suggested.
Generating optimized autoload files
86 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
No security vulnerability advisories found

AzuraCast Restore
=================



 [ERROR] Backup-Pfad /var/azuracast/backups/automatic_backup_20221221_020621.zip nicht gefunden!


[2022-12-21T22:29:55.086451+00:00] AzuraCast.WARNING: Console command `azuracast:restore` exited with                                                                                                              error code 1. [] []
mv: can't rename '/backup_src/automatic_backup_20221221_020621.zip': No such file or directory
[+] Running 1/1
 ⠿ Network azuracast_default  Removed                                                            0.1s
WARN[0000] volume "azuracast_backups" already exists but was not created by Docker Compose. Use `exter                                                                                                             nal: true` to use an existing volume
[+] Running 2/2
 ⠿ Network azuracast_default  Created                                                            0.0s
 ⠿ Container azuracast        Started
```

**Comment by ItsMitchh** [comment]
> Dunno why azuracast uses /var/lib/docker/volumes/ instead of /var/azuracast/
AzuraCast has always used this method, it's how we use Docker and how Docker works. 

We do explicitly say we'll remove all existing files: 
```Restoring will remove any existing AzuraCast installation data, replacing it with your backup. Continu                                                                                                             e? [Y/n] Y```

**Comment by Schrebergarten03-links** [comment]
hm well okay my fault ... 

But still the default backup location doesnt makes sense when it deletes itself and every other backup ... imho that should not be standart setting ... 

Well i learned the hard way.

**Comment by LordHelmchen666** [comment]
Thats the reason why i use docker-compose.override.yaml to save Backups not in Docker. 
```
services:
 web:
  volumes:
      - /private-backup/azuracast:/var/azuracast/backups
```
I also rsync the backups  to another server. I run a cronjob on my backupserver once a day
`rsync -avzhe ssh --delete root@ip_azuracast_server:/private-backup/azuracast/ /home/backups/azuracast_zip/`

**Comment by Schrebergarten03-links** [comment]
Ah thank you im gonna do that now as well :)

**Comment by fnfzlznddzn** [comment]
I can't really help with the backup situation, but I would strongly advise you to make your playlists completely independent from azuracast, i.e. one folder for each playlist, in a directory outside of azuracast, and tie the folder to an azuracast playlist (it will be watched and updated). This way you can recreate the playlists without the pain of re-selecting the songs if a problem happens.
Good luck!

**Comment by dvdokkum** [comment]
Yeah this is what we do. Playlists watch a folder... that way you never have to fiddle too much with the playlists themselves.

I've also learned about self-deleting backups the hard way @Schrebergarten03-links. I have since adopted the same approach as @LordHelmchen666... alias a folder into the docker file system and have backups save there. Would also recommend taking regular snapshots outside of the AC backup system if you're not already doing that.
